jvm/controls/src/main/resources/css/add-plugin-dialog.css

```diff
@@ -46,16 +45,16 @@
 }
 
 .add-plugin-dialog__btn--close:hover {
-    -fx-background-color: -wa-gray-lighter;
+    -fx-background-color: -wa-foreground-hover;
```

Comment:
Why not -wa-background-hover, since it's giving the color to the background of the button? If it's taken for other purposes then you can give it a more specific name, like -wa-button-background-hover